### PR TITLE
Confirm removing project in mystuff

### LINKED ISSUE #4884: confirm-actions: When leaving a studio in /mystuff, confirmation modal opens
### Description
When you try to leave a studio in my stuff, it shows a modal meant for the unsharing projects feature.

### Reproduction steps

1. Enable confirmations addon, enable "Project unsharing" setting
2. Go to /mystuff
3. Try to leave/delete a studio
4. See modal

### Expected behavior
Forcefully leave the studio

### Actual behavior
Opens modal

### Screenshots

https://user-images.githubusercontent.com/100447465/180599462-a8a4dc93-c2c8-458b-9c23-af63ca27da70.mp4

### Environment

- Browser: Firefox 
- Scratch Addons version: 1.27.3
- Operating system: Windows 11

### Additional context
We could turn this into a feature?

### PR DESCRIPTION
**Pull request by Norbiros**
Resolves #4884

### Changes

- Fix bug with showing "unshare project" when leaving/deleting studio in mystuff
- Show new delete confirmation when user is deleting project in mystuff (in trash, and in projects tab)

### Tests

Firefox 95.

### CODE REVIEW

**Comment by mxmou** [comment]
studio-tools already shows a confirmation for leaving studios. It doesn't have to be duplicated in this addon.

**Comment by Norbiros** [comment]
Oh, okay... Fixed

File: addons-l10n/en/confirm-actions.json [hunk 1]
```diff
@@ -10,5 +10,7 @@
   "confirm-actions/closetopic-title": "Close Topic",
   "confirm-actions/closetopic": "Are you sure you want to close this topic?",
   "confirm-actions/cancelcomment-title": "Discard Comment",
-  "confirm-actions/cancelcomment": "Are you sure you want to discard your comment?"
+  "confirm-actions/cancelcomment": "Are you sure you want to discard your comment?",
+  "confirm-actions/removeproject-title": "Remove Project",
+  "confirm-actions/removeproject": "Are you sure you want to remove this project?"
```
Comment (mxmou) [inline addons-l10n/en/confirm-actions.json:15]:
```suggestion
  "confirm-actions/removeproject-title": "Delete Project",
  "confirm-actions/removeproject": "Are you sure you want to delete this project?"
```

File: addons/confirm-actions/addon.json [hunk 2]
```diff
@@ -53,11 +53,17 @@
       "id": "closingtopic",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Confirm removing projects",
+      "id": "removingprojects",
+      "type": "boolean",
+      "default": false
     }
```
Comment (mxmou) [inline addons/confirm-actions/addon.json:62]: Rename this to "Confirm deleting projects" and move it after "Confirm project unsharing".

I think it should be enabled by default.

File: addons/confirm-actions/userscript.js [hunk 3]
```diff
@@ -11,14 +11,15 @@ export default async function ({ addon, console, msg }) {
 
       let title = null;
       let cancelMessage = null;
+      let isInStudios = document.querySelector("#tabs > :nth-child(4).active");
```
Comment (mxmou) [inline addons/confirm-actions/userscript.js:14]: I think it's better to check `location.hash`.

**Review by mxmou — CHANGES_REQUESTED** [review]

**Review by mxmou — APPROVED** [review]

**Review by apple502j — APPROVED** [review]

**Pull request merged**